dell-r730xd/workflows/dell-perccli-delete-megaraid-graph.json

```diff
@@ -33,21 +38,36 @@
             "label": "delete-raid",
             "taskName": "Task.Raid.Delete.MegaRAID",
             "waitOn": {
-                "bootstrap-ubuntu": "succeeded"
+                "reboot": "succeeded"
             }
         },
         {
+            "ignoreFailure": true,
             "label": "refresh-catalog-megaraid",
             "taskName": "Task.Catalog.perccli",
             "waitOn": {
                 "delete-raid": "succeeded"
             }
         },
         {
+            "label": "refresh-catalog-driveid",
+            "taskName": "Task.Catalog.Drive.Id",
+            "waitOn": {
+                "refresh-catalog-megaraid": "succeeded"
+            }
+        },
+        {
+            "label": "finish-bootstrap-trigger",
+            "taskName": "Task.Trigger.Send.Finish",
+            "waitOn": {
+                "final-reboot": "finished"
```

Comment:
If this task waitOn final-reboot, you'd better to move it to be after final-reboot